README.md

```diff
@@ -0,0 +1,21 @@
+# loke-http-rpc-client
+
+Client library for `loke-http-rpc` server library.
+
+## Tools
+
+### download-rpc-meta
+
+Run:
+
+```
+npx download-rpc-meta {name_of_service} {url_to_rpc_server}
+```
+
+### generate-rpc-tsd
+
+Analyses files in the ipc_manifests folder and creates .d.ts files.
+
+```
+npx generate-rpc-tsd
```

Comment:
Maybe we could just make this happen automatically as part of `download-rpc-meta`